tests/test_browser.py

```diff
@@ -3400,13 +3400,13 @@ def test_modularize_network_error(self):
             reportResultToServer("Module creation succeeded when it should have failed");
           })
           .catch(err => {
-            reportResultToServer(err.message.slice(0, 54));
```

Comment:
I think this was to avoid huge output in the log?

Reply:
Right, but I eliminated that problem by no longer including the stack track in the message itself.   That inclusion of the stack trace embedded in the message was one of the causes of the duplication.   Since we don't do that anymore there messages are no longer excessively long.

Reply:
Oh ok, sgtm.